Client/Renderer.cpp

```diff
@@ -21,7 +21,7 @@ GameTechRenderer::GameTechRenderer(GameWorld& world, Canvas& canvas) : OGLRender
     debugShader  = new OGLShader("debug.vert", "debug.frag");
     shadowShader = new OGLShader("shadow.vert", "shadow.frag");
     textShader = std::make_shared<OGLShader>("text.vert", "text.frag");
-    defaultShader = new OGLShader("scene.vert", "Buffer.frag");
```

Comment:
Why has their been a whole shader change for deferred rendering?

Reply:
Acc to matthew:  in his graphics coursework he had his def rendering this fragment shader so he had that in the project. now that hes put it in scene.frag this shouldbe ok to remove